mdx-web/src/main/java/com/mx/path/model/mdx/web/MdxWebApplicationException.java

```diff
@@ -0,0 +1,13 @@
+package com.mx.path.model.mdx.web;
+
+import com.mx.common.exception.PathSystemException;
+
+public class MdxWebApplicationException extends PathSystemException {
```

Comment:
Sorry if this is a dumb question, but would any of these ever be thrown for a client like AFCU? Just wondering if I should let them know in case they see any exceptions changing on their end.

Reply:
Exceptions here would stop the application from booting and would require us to fix them. They won't surface to AFCU, and this hopefully wouldn't ever happen in int or prod.

Reply:
I think this exception will only be used by the mdx-web code if there's an error starting up the application, so if we have that kind of exception we'd have bigger issues and they wouldn't surface to AFCU.

Reply:
An added bonus... k8s will not bring down the functioning pods if we accidentally borked an application so bad that it wouldn't startup. So, a failure in this class, theoretically (😆 ), wouldn't bring the application down.

🎉

Reply:
Nice! Okay, this is good to know, thanks!